lib/active_admin/page.rb

```diff
@@ -84,10 +84,11 @@ def clear_page_actions!
       @page_actions = []
     end
 
-    def belongs_to(target, options = {})
-      @belongs_to = Resource::BelongsTo.new(self, target, options)
-      self.navigation_menu_name = target unless @belongs_to.optional?
-      controller.send :belongs_to, target, options.dup
+    def belongs_to(targets, options = {})
+      @belongs_to = Resource::BelongsTo.new(self, targets, options.dup)
```

Comment:
Same question about mutating the options.

Reply:
Same.

Reply:
Let's leave it as it is if not necessary.